# HG changeset patch
# User Rik <rik@octave.org>
# Date 1407683898 25200
#      Sun Aug 10 08:18:18 2014 -0700
# Node ID 391e080ae81061e48d29edeff9b2dff170101d9b
# Parent  1ad621d894bac1cd6207d47bc504b57403171e11
__go_draw_axes__.m: Fix regression using legend with log axes introduced in f84361735a40.

* __go_draw_axes__.m: Use line continuation when creating fontspacespec so that wrapping
line to < 80 characters doesn't produce a 2 row matrix.

diff --git a/scripts/plot/util/private/__go_draw_axes__.m b/scripts/plot/util/private/__go_draw_axes__.m
--- a/scripts/plot/util/private/__go_draw_axes__.m
+++ b/scripts/plot/util/private/__go_draw_axes__.m
@@ -1585,17 +1585,17 @@ function __go_draw_axes__ (h, plot_strea
         warning ("legend: 'Best' not yet implemented for location specifier.\n");
         ## Least conflict with data in plot.
         ## Least unused space outside plot.
       otherwise
         pos = "";
     endswitch
     if (__gnuplot_has_feature__ ("key_has_font_properties"))
       [fontname, fontsize] = get_fontname_and_size (hlgnd);
-      fontspacespec = [ create_spacingspec(fontname, fontsize, gnuplot_term),
+      fontspacespec = [ create_spacingspec(fontname, fontsize, gnuplot_term),...
                         create_fontspec(fontname, fontsize, gnuplot_term) ];
     else
       fontspacespec = "";
     endif
     textcolors = get (findobj (hlgnd.children, "type", "text"), "color");
     if (iscell (textcolors))
       textcolors = cell2mat (textcolors);
       textcolors = unique (textcolors, "rows");
